Prox/Prox/Utilities/Animatable.swift

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/// A protocol for to adhere to for anything that we want to animate
+protocol Animatable {
```

Comment:
An interface I made to return back animation property structs so I don't need to expose views/variables in the package space when I don't need to just for animations.